core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -93,12 +93,14 @@ public static void initializeNatives(Path tmpFile, boolean mlockAll, boolean sec
                 throw new RuntimeException("don't run elasticsearch as root.");
             }
         }
-        
+
         // enable secure computing mode
         if (seccomp) {
             Natives.trySeccomp(tmpFile);
+        } else {
+            logger.warn("Not enabling seccomp because it was explicitly disabled");
         }
```

Comment:
I just think we can launch a little perl script in pre-integration-test, kill it in post-integration-test, that sits on a socket and starts up nodes. Its a perl script for testing. Then the rest of the stuff can run without us having to defeat every security mechanism we are trying to add...